parts/preface/introduksjon.tex

```diff
@@ -7,29 +7,29 @@ \chapter*{Forord}
 	\label{fig:arne-hassel}
 \end{figure}
 
-Cybernetisk Selskab er en formidabel forening på Institutt for informatikk. I femti år har foreningen jobbet for å bedre forholdene for studenter, være det for dem gikk linjen Kybernetikk eller for alle studenter på instituttet. Hva dette har betydd i praksis har endret seg gjennom tidene, fra å være rettet mot mer faglig og studentpolitisk innhold, til mer sosialt og kulturbyggende.
+Cybernetisk Selskab (CYB) er en formidabel forening på Institutt for informatikk (Ifi). I femti år har foreningen jobbet for å bedre forholdene for studenter, være det for dem som gikk linjen Kybernetikk eller for alle studenter på instituttet. Hva dette har betydd i praksis har endret seg gjennom tidene, fra å være rettet mot mer faglig og studentpolitisk innhold, til mer sosialt og kulturbyggende.
 
-Denne boken er et forsøk på å beskrive historien til CYB, Ifi og studentmiljøet på instituttet. Hovedvekten ligger på førstnevnte, dette er tross alt CYB sin jubileumsbok, men det blir en mangelfull helhet av CYB om man ikke også inkluderer det miljøet som foreningen har vært del av.
+Denne boken er et forsøk på å beskrive historien til CYB, Ifi og studentmiljøet på instituttet. Hovedvekten ligger på førstnevnte: dette er tross alt CYB sin jubileumsbok, men det blir en mangelfull helhet av CYB om man ikke også inkluderer det miljøet som foreningen har vært del av.
```

Comment:
Kanskje dette er et sted hvor det faktisk er innenfor å bruke semikolon?

Reply:
Jeg tenkte på det. Kolon kan brukes som forklaring av det som kommer før, men jeg er usikker på om det funker med resten av leddsetningen.

Reply:
Semikolon ser ikke ut til å være riktig å ta i bruk her ([The Oatmeal FTW](https://www.google.com/imgres?imgurl=https://s3.amazonaws.com/theoatmeal-img/comics/semicolon/super.png&imgrefurl=https://theoatmeal.com/comics/semicolon&h=872&w=662&tbnid=FmbB5rLRgULlEM:&q=when+do+you+use+semicolons&tbnh=160&tbnw=121&usg=AI4_-kShP2CVNIIYg4szDCGrsMTg_EKDBQ&vet=12ahUKEwjyoJDez5bgAhVI_aQKHcIpCdkQ9QEwAHoECAgQBg..i&docid=K7CvAx4W52V8wM&sa=X&ved=2ahUKEwjyoJDez5bgAhVI_aQKHcIpCdkQ9QEwAHoECAgQBg)). Vi lar det være med kolon.